README.md

```
@@ -1,2 +1,117 @@
+
+</aside>
+
+
+## 💬 배경 Background 
+
+![Slide 16_9 - 1062](https://github.com/user-attachments/assets/4cc68116-88a5-4261-a079-c9de7a9c5c4b)
+>  최근 영화진흥위원회가 공개한 “2024년 한국 영화산업 결산’에 따르면, 2019년 약 1조 9천억 원에 달했던 극장 매출은 코로나19 이후 급감했고, 2024년에도 약 1조 1천억 원 수준에 머무르며 약 38% 감소한 것으로 나타나, 완전한 회복까지는 여전히 어려운 상황이다.
+
+### 🎥 영화관의 활용 변화
+
+**복합문화공간으로 바뀌는 영화관의 용도**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)